<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>511</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Nuevo producto</string>
     </attribute>
     <widget class="QLineEdit" name="Txtcodigo">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TxtDescripcion">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>33</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Codigo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>82</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Descripcion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Precio</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Unidad</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TxtPrecio">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TxtUnidad">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnGuardar">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnEditar">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnEliminar">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Listado de producto</string>
     </attribute>
     <widget class="QTableWidget" name="Tbl_productos">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>491</width>
        <height>331</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Codigo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Descripcion</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Precio</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Unidad</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
